src/test/java/org/mule/tools/apikit/output/MuleConfigGeneratorTest.java

```diff
@@ -143,4 +145,49 @@ public void blankDocumentWithoutLCInDomain() {
     assertEquals("/console/*", consoleFlow.getChildren().get(0).getAttribute("path").getValue());
     assertEquals("console", consoleFlow.getChildren().get(1).getName());
   }
+
+  @Test
+  public void blankDocumentWithoutLCInDomainHideConsole() {
```

Comment:
This is almos a copy from the test above. Try to subtract the common code to a parameterized method and leave the particular asserts on each test.